pkg/cfg/config.go

```diff
@@ -74,6 +74,21 @@ func NewWithInterfaces(lookupEnv LookupEnv) *config {
 	return cfg
 }
 
+// load and merge multiple configuration files
+func FromFiles(filePaths []string, fileType string) GosoConf {
```

Comment:
👍 Cool I can use this also in my branch. 

Just a small suggestion. If you change the order of the arguments you can make this a variadic function which would be nicer to use.

Reply:
Please don't extend the interface of the config..
We're using the option pattern here. Please try to stick with that..

Reply:
@j4k4 can we define some rules when convenience functions are allowed and when not?

Reply:
It's hard to define rules for this kind of stuff..
Actually, at first i read this one wrong and thought this function would have the Config as a receiver.. As this is a convenience function, it's not as bad as i thought..
Nevertheless, i wouldn't put it into the cfg package as this function is based on your use case in the test package? So it would be better to have this in there?